scripts/deploy_website.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+JSON_FILE=../cookiecutter.json
+YAML_FILE=../cookiecutter.yaml
+
+if [ -f "$JSON_FILE" ]; then
+   rm "$JSON_FILE"
+   echo "Removed JSON file"
+fi
+
+if [ -f "$YAML_FILE" ]; then
+   echo "Converting yaml to json"
+   python yaml2json.py "$YAML_FILE" "$JSON_FILE"
+fi
+
+echo "Building the jupyter book"
+jupyter-book build ../book/
+
+echo "building the splash page"
+
+if [ -d "../book/_build/html/assets" ]; then
+   rm -rf ../book/_build/html/assets
+fi
+  
+cookiecutter ../. -f --no-input -o ../book/_build
+
```

Comment:
I'm guessing we ultimately don't want the splash page assets and `cookiecutter.json` file actually saved to the repo. I'm going to add them to the `.gitignore` so that they shouldn't accidentally get pushed by a `git add all`.

Reply:
Also, I added a line to this script to cd into it's local directory so that the relative paths work - I tried running it from the top of the repo and it couldn't find the relative paths. @aaarendt can you confirm it still works for you?

Reply:
@JessicaS11 my understanding is the new `index.html` file refers to the `assets/` folder and needs to be uploaded to the repo/branch in which this ultimately gets deployed.

Reply:
Good idea to include the script for changing to local directory. I tried that and it works!